wiki/osu!_wiki/Contribution_guide/Best_practices/de.md

```diff
@@ -35,27 +30,17 @@ Wenn du während der Verwendung des GitHub-Tools auf irgendwelche Probleme stö
 
 1. Öffne **deine Verzweigung** und gehe zum Tab `Actions`.
```

Comment:
BTW: Translating `fork` as `Verzweigung` should be a crime. Out of scope for this PR though